### PR TITLE
remove all suri settings from argo (no longer used)

### PR DESCRIPTION
**Pull request by ndushay**
## Why was this change made?

To reduce confusion about suri configuration settings

## How was this change tested?

test suite

## Which documentation and/or configurations were updated?

shared_configs for qa, for stage, config files in app (see Files changed), and a PR for shared_configs for prod:  please merge sul-dlss/shared_configs/pull/1381 after this is merged.

### CODE REVIEW

File: config/initializers/dor_config.rb [hunk 1]
```diff
@@ -17,12 +17,9 @@
     url Settings.solrizer_url
   end
 
+  # SURI url currently only used for okcomputer;  it's essential for registering objects
```
Comment (jcoyne) [inline config/initializers/dor_config.rb:20]: I think we can remove the OKComputer check too. There is no direct dependency.

**Comment by jcoyne** [comment]
It looks like a druid needs to be provided to `spec/features/item_view_metadata_spec.rb`

File: config/initializers/dor_config.rb (hunk 1 above)
Comment (ndushay) [reply config/initializers/dor_config.rb:20]: will do.

**Comment by ndushay** [comment]
Thanks, Justin.  I was hoping for the failing spec to magically go away, but a great steer works, too.  :-)

**Comment by jcoyne** [comment]
We know about great steer in Minnesota

File: spec/features/item_view_metadata_spec.rb [hunk 2]
```diff
@@ -125,6 +125,7 @@
       let(:dro_struct) { instance_double(Cocina::Models::DROStructural, contains: []) }
 
       before do
+        allow(item).to receive(:pid).and_return('fc123ky4567')
```
Comment (jcoyne) [inline spec/features/item_view_metadata_spec.rb:128]: I'd suggest fixing it by providing the pid to the item (with the druid: prefix).

**Comment by ndushay** [comment]
this is ready for re-review;  after merging, please merge sul-dlss/shared_configs/pull/1381

**Pull request merged**